packages/react-instantsearch/src/components/Highlight.js

```diff
@@ -1,19 +1,6 @@
 import React from 'react';
+import Highlighter from './Highlighter';
 
-export default function Highlight({hit, attributeName, highlight}) {
-  const parsedHighlightedValue = highlight({hit, attributeName});
-  const reactHighlighted = parsedHighlightedValue.map((v, i) => {
-    const key = `split-${i}-${v.value}`;
-    if (!v.isHighlighted) {
-      return <span key={key} className="ais-Highlight__nonHighlighted">{v.value}</span>;
-    }
-    return <em key={key} className="ais-Highlight__highlighted">{v.value}</em>;
-  });
-  return <span className="ais-Highlight">{reactHighlighted}</span>;
+export default function Highlight(props) {
+  return <Highlighter path="_highlightResult" {...props}/>;
```

Comment:
Yes. The path is not a prop documented in the Highlight/Snippet widget so I figured out it was ok to just forward the props. Do you think it's an issue?